package.json

```diff
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/Kodaskills/kodaskills.co#readme",
   "description": "",
   "dependencies": {
-    "astro": "^5.7.11"
+    "astro": "^5.7.12"
```

Comment:
@williamgentil be careful to add packages dependencies changes into their own commit. It's useful to know when we change packages on logs because often it's the main problem when a bug raise.